src/Amalgam/entity/EntityQueryBuilder.h

```diff
@@ -39,84 +39,84 @@ namespace EntityQueryBuilder
-			[&dist_params](size_t i, bool found, EvaluableNode *en) {
-				if(i < dist_params.featureParams.size())
+			[&dist_eval](size_t i, bool found, EvaluableNode *en) {
+				if(i < dist_eval.featureAttribs.size())
 				{
 					if(found)
-						dist_params.featureParams[i].weight = EvaluableNode::ToNumber(en);
+						dist_eval.featureAttribs[i].weight = EvaluableNode::ToNumber(en);
 					else
-						dist_params.featureParams[i].weight = 1.0;
+						dist_eval.featureAttribs[i].weight = 1.0;
 				}
 			});
 
 		//get type
 		EvaluableNode::ConvertChildNodesAndStoreValue(distance_types_node, element_names, num_elements,
-			[&dist_params](size_t i, bool found, EvaluableNode *en) {
-				if(i < dist_params.featureParams.size())
+			[&dist_eval](size_t i, bool found, EvaluableNode *en) {
+				if(i < dist_eval.featureAttribs.size())
 				{
-					auto feature_type = GeneralizedDistance::FDT_CONTINUOUS_NUMERIC;
+					auto feature_type = GeneralizedDistanceEvaluator::FDT_CONTINUOUS_NUMERIC;
 					if(found)
 					{
 						StringInternPool::StringID feature_type_id = EvaluableNode::ToStringIDIfExists(en);
 						switch(feature_type_id)
 						{
-						case ENBISI_nominal_numeric:			feature_type = GeneralizedDistance::FDT_NOMINAL_NUMERIC;			break;
-						case ENBISI_nominal_string:				feature_type = GeneralizedDistance::FDT_NOMINAL_STRING;				break;
-						case ENBISI_nominal_code:				feature_type = GeneralizedDistance::FDT_NOMINAL_CODE;				break;
-						case ENBISI_continuous_numeric:			feature_type = GeneralizedDistance::FDT_CONTINUOUS_NUMERIC;			break;
-						case ENBISI_continuous_numeric_cyclic:	feature_type = GeneralizedDistance::FDT_CONTINUOUS_NUMERIC_CYCLIC;	break;
-						case ENBISI_continuous_string:			feature_type = GeneralizedDistance::FDT_CONTINUOUS_STRING;			break;
-						case ENBISI_continuous_code:			feature_type = GeneralizedDistance::FDT_CONTINUOUS_CODE;			break;
-
-						default:								feature_type = GeneralizedDistance::FDT_CONTINUOUS_NUMERIC;			break;
+						case ENBISI_nominal_numeric:			feature_type = GeneralizedDistanceEvaluator::FDT_NOMINAL_NUMERIC;			break;
+						case ENBISI_nominal_string:				feature_type = GeneralizedDistanceEvaluator::FDT_NOMINAL_STRING;				break;
+						case ENBISI_nominal_code:				feature_type = GeneralizedDistanceEvaluator::FDT_NOMINAL_CODE;				break;
+						case ENBISI_continuous_numeric:			feature_type = GeneralizedDistanceEvaluator::FDT_CONTINUOUS_NUMERIC;			break;
+						case ENBISI_continuous_numeric_cyclic:	feature_type = GeneralizedDistanceEvaluator::FDT_CONTINUOUS_NUMERIC_CYCLIC;	break;
+						case ENBISI_continuous_string:			feature_type = GeneralizedDistanceEvaluator::FDT_CONTINUOUS_STRING;			break;
+						case ENBISI_continuous_code:			feature_type = GeneralizedDistanceEvaluator::FDT_CONTINUOUS_CODE;			break;
+
+						default:								feature_type = GeneralizedDistanceEvaluator::FDT_CONTINUOUS_NUMERIC;			break;
 						}
```

Comment:
The `break`s here don't show up as aligned in VSCode diff. It isn't clear if this is just a diff thing or the tabs or off in the formatting...